backend/src/main/java/com/huellapositiva/domain/exception/TimeForRecoveringPasswordExpiredException.java

```diff
@@ -0,0 +1,7 @@
+package com.huellapositiva.domain.exception;
+
+public class TimeForRecoveringPasswordExpiredException extends RuntimeException {
```

Comment:
If you annotate this class with @ResponseStatus you won't need to catch this exception in the controller. Up to you

Reply:
We changed it, but we have some doubts about which way is more adequate?

Reply:
It is up to you how much Spring "magic" you want to use in your code. Under a design perspective you should not use the magic since if couples your code to the concrete framework that is performing that magic, makes units less testable and hides part of the logic to mention a few drawbacks.

There are lots of cons/pros around the topic. My personal taste is to use the magic as long as I know what that magic does behind the scenes and does not influence in my testing strategy. That allows a very fast development lifecycle once you know what you are doing,  but would not recommend it to someone that is getting to know Spring framework and overall application development.